### ISSUE TITLE
Add Alt Text to Contributor Avatars

### ISSUE THREAD

**Issue by CodeMaverick-143** [XplnHUB/xplnhub#10]
The img tag for contributor avatars in src/pages/Contributors.tsx uses the login as the alt text. This should be more descriptive, for example, "Avatar of {contributor.login}.

**Comment by Prabhav1437** [comment]
completed please close it
